### PR TITLE
Adding YAML Test Cases - Nov 2nd

### PR DESCRIPTION
**Pull request by kvikrambhat**
**Test Script Added**
- Adding YAML test scripts for
1.Test_TC_BI_2_1
2.Test_TC_CC_9_2
3.Test_TC_CC_9_3
4.Test_TC_OCC_2_2
5.Test_TC_LVL_4_1
6.Test_TC_LVL_5_1
7.Test_TC_PRS_1_1
8.Test_TC_PRS_2_1
9.Test_TC_RH_2_2
10.Test_TC_TM_2_2
11.Test_TC_TSUIC_2_2
12.Test_TC_WNCV_2_5
13.Test_TC_DIAGSW_1_1
- Generated script files

**Testing**
Executed all YAML test scripts, attached logs for reference
[Complete_Test_Run.txt](https://github.com/project-chip/connectedhomeip/files/7465990/Complete_Test_Run.txt)

### CODE REVIEW

**Comment by jmeg-sfy** [comment]
Looks like you have dropped the previous PR ?

File: src/app/tests/suites/certification/Test_TC_BI_2_1.yaml [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+name: 18.2.1. [TC-BI-2.1] Attributes with server as DUT
+
+config:
+    cluster: "Binary Input (Basic)"
+    endpoint: 1
+
+tests:
+    - label: "Read mandatory non-global attribute: OutOfService"
+      command: "readAttribute"
+      attribute: "out of service"
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_BI_2_1.yaml:24]: I Prefer all CamelCase style -> this should follow the specs and the implementation

File: src/app/tests/suites/certification/Test_TC_BI_2_1.yaml [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+          "Write the default values to mandatory non-global attribute:
+          PresentValue"
+      command: "writeAttribute"
+      attribute: "present value"
+      arguments:
+          value: 0
+
+    - label: "Reads back the mandatory non-global attribute: PresentValue"
+      command: "readAttribute"
+      attribute: "present value"
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_BI_2_1.yaml:66]: Same

File: src/app/tests/suites/certification/Test_TC_LVL_4_1.yaml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 3000
+
+    - label: "Reads current level attribute from DUT"
+      command: "readAttribute"
+      attribute: "current level"
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_LVL_4_1.yaml:51]: CamelCase

**Review by jmeg-sfy — APPROVED** [review]
If all tests are passing let go.

**Comment by kvikrambhat** [comment]
> Looks like you have dropped the previous PR ? #11305

Yes.

File: src/app/tests/suites/certification/Test_TC_BI_2_1.yaml (hunk 1 above)
Comment (kvikrambhat) [reply src/app/tests/suites/certification/Test_TC_BI_2_1.yaml:24]: > If all tests are passing let go.

Attribute names sometimes don't match the spec's convention, we are using the names the YAML framework spits out when it fails to find any attribute.

**Comment by kvikrambhat** [comment]
> If all tests are passing let go.

In some cases the attribute names don't match the spec's convention, we are using the names that the YAML framework spits out when attribute names are not reionized by it.

**Review by woody-apple — APPROVED** [review]

**Comment by woody-apple** [comment]
Fast tracking, given tests

**Review by andy31415 — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_BI_2_1.yaml (hunk 1 above)
Comment (kvikrambhat) [reply src/app/tests/suites/certification/Test_TC_BI_2_1.yaml:24]: Nothing to be done currently, Test Script are passing and naming conventions are as per SDK.

File: src/app/tests/suites/certification/Test_TC_BI_2_1.yaml (hunk 2 above)
Comment (kvikrambhat) [reply src/app/tests/suites/certification/Test_TC_BI_2_1.yaml:66]: Nothing to be done currently, Test Script are passing and naming conventions are as per SDK.

File: src/app/tests/suites/certification/Test_TC_LVL_4_1.yaml (hunk 3 above)
Comment (kvikrambhat) [reply src/app/tests/suites/certification/Test_TC_LVL_4_1.yaml:51]: Nothing to be done currently, Test Script are passing and naming conventions are as per SDK.

**Pull request merged**